lai_textclf/utilities.py

```diff
@@ -0,0 +1,19 @@
+import warnings
+
+from lightning.app.storage import Drive
+from lightning.app.utilities.cloud import is_running_in_cloud
+
+
+def warn_if_drive_not_empty(drive: Drive):
+
+    if drive.list():
+        warnings.warn(
+            "Drive is not empty! This may result into wrong logging behaviour if your app doesn't have a built-in resume mechanism. Consider deleting the .lightning file and restarting the app."
```

Comment:
```suggestion
            "Drive is not empty! This may result in wrong logging behaviour if your app doesn't have a built-in resume mechanism. Consider deleting the .lightning file and restarting the app."
```
How is deleting .lightning helping here? Is it because of the name of the app? I would probably rather suggest to choose a different name via --name in the CLI :)

Reply:
edit: added both options here